### ISSUE TITLE
A11y - OU: Lacking descriptive page titles

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5529]
All pages must have a {{<title>}} that describes the topic or purpose. Current storefront pages follow this pattern: 
{{<name of page> | Demostore}}

expected: {{Organization Units | Demostore}}

actual: {{Demostore}}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39690**



[created from NEXT-39690, comment 513926]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39690, comment 519892]
